apps/taxes/src/lib/app/redirect.ts

```diff
@@ -0,0 +1,43 @@
+import { useAppBridge } from "@saleor/app-sdk/app-bridge";
+import { AppLinkProps } from "../../modules/ui/app-link";
+
+export const useAppRedirect = () => {
+  const { appBridge } = useAppBridge();
+
+  const redirectToPath = (props: AppLinkProps) => {
+    return appBridge?.dispatch({
+      type: "redirect",
+      payload: {
+        newContext: false,
+        actionId: "redirect_from_tax_app",
+        to: props.path ? props.path : props.href,
+      },
+    });
+  };
+
+  return { redirect: redirectToPath };
+};
+
+const paths = {
+  taxConfiguration: `/taxes/channels`,
+} as const;
+
+export type DashboardPath = keyof typeof paths;
+
+type DashboardRedirectValues = [AppLinkProps, () => void];
+
+export const useCreateDashboardRedirect = (p: DashboardPath): DashboardRedirectValues => {
+  const { appBridgeState } = useAppBridge();
+  const { redirect } = useAppRedirect();
+
+  const domain = appBridgeState?.domain;
+  const path = paths[p];
+  const href = `https://${domain}/dashboard${path}`;
```

Comment:
Do we have access to the parent window? If so, that's going to be a better source of the URL.
Assuming the "https" might cause problems on fully local setups. Alternative approach (change localhost to ApiUrl):
```
new URL("/dashboard", "http://localhost:8000/graphql/").href
'http://localhost:8000/dashboard'
```

Reply:
`/taxes/channels` doesn't work, `/dashboard/taxes/channels` does. But overall, I agree.